client/src/routes/human_resources/meal_tickets/index.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import Breadcrumb from '../../../components/Breadcrumb.svelte';
+  import { _ } from 'svelte-i18n';
+  import H4Title from '../../../components/H4Title.svelte';
+  import { onMount } from 'svelte';
+  import { errorNormalizer } from '../../../normalizer/errors';
+  import ServerErrors from '../../../components/ServerErrors.svelte';
+  import { get } from '../../../utils/axios';
+  import Table from './_Table.svelte';
+  let title = $_('human_resources.meal_tickets.title');
+
+  let mealTicketsSummarries = [];
+  let errors;
+
+  const fetchMealTicketsSummaries = async () => {
+    try {
+      mealTicketsSummarries = (await get('meal-tickets/count')).data;
+    } catch (e) {
+      errors = errorNormalizer(e);
+    }
+  };
+
+  onMount(async () => {
+    await fetchMealTicketsSummaries();
+  });
+</script>
+
+<svelte:head>
+  <title>{title} - {$_('app')}</title>
+</svelte:head>
+
+<Breadcrumb
+  items="{[{ title: $_('human_resources.breadcrumb') }, { title }]}" />
+<H4Title title="{$_('human_resources.meal_tickets.available_meal_tickets')}" />
+
+<ServerErrors errors="{errors}" />
```

Comment:
pourquoi pas juste 
```suggestion
<ServerErrors errors={errors} />
```
?

Reply:
> pourquoi pas juste
> 
> ?

Il y a pas de  raison technique particulière c'est juste la config de prettier/eslint qui est configurée comme ça.

Ca se change. Par exemple sur vimeet365 c'est la forme raccourcie qui est privillégiée

Exemple : 

```
<ServerErrors {errors} />
```

Si tu regardes les autres fichier de permacoop ils sont formatté avec cette forme.

```
<ServerErrors errors="{errors}" />
```

Reply:
`<ServerErrors {errors} />` serait plus "propre" je pense

Reply:
On a déja eu cette discussion il y a quelques mois je pense.

Actuellement sur le projet les autres composant sont formatter sans utiliser la notation raccourcie  (shorthand)

```
<ServerErrors errors="{errors}" />
```

Si il est préférable pour toi de formatter les composant avec la forme raccourcie alors je propose de prendre en compte le format  aussi dans les autres composants.

C'est l'objet de cette PR https://github.com/fairnesscoop/permacoop/pull/203

Perso, je te le disais déja. Je trouve ça un peu dommage que l'on passe du temps à discuter de formattage et de problématiques de CS alors que des outils comme eslint ou prettier permettent de garder une cohérence et de se mettre d'accord sur un format.